ui/search.js

```diff
@@ -53,17 +58,48 @@ function removeCloseButton() {
     delete searchResults._closeButton;
 }
 
-function setSearchResultsXAlign(align) {
+function addMaxWidthBoxInSearch() {
     const { viewSelector } = Main.overview;
-    viewSelector._searchResults.x_align = align;
+
+    viewSelector._searchPage.remove_child(viewSelector._searchResults);
+
+    const maxWidthBox = new Search.MaxWidthBox({ name: 'searchResultsBox' });
+    maxWidthBox.add_child(viewSelector._searchResults);
```

Comment:
Not sure I understand how this guarantees the search box will always be the same size, care to elaborate?

Reply:
Setting the x_align to CENTER makes the default allocators use the minimum if necessary width. Search wants as much as possible, but not more than `max-width`

Reply:
Ok.